packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/ova/ovaProviderValidator.ts

```diff
@@ -7,11 +7,11 @@ export function ovaProviderValidator(provider: V1beta1Provider): ValidationMsg {
   const url = provider?.spec?.url || '';
 
   if (!validateK8sName(name)) {
-    return { type: 'error', msg: 'invalided kubernetes resource name' };
+    return { type: 'error', msg: 'invalid kubernetes resource name' };
   }
 
   if (!validateNFSMount(url)) {
-    return { type: 'error', msg: 'invalided OVA mount end point' };
+    return { type: 'error', msg: 'invalid OVA mount end point' };
```

Comment:
```suggestion
    return { type: 'error', msg: 'invalid OVA mount endpoint' };
```

I not an English expert, but I think that `endpoint` is a one-word term.

Reply:
thanks fixing